### ISSUE TITLE
On Hover, Show that Spacebar is used to Open the Objects' Menu

### ISSUE THREAD

**Issue by flowpoly** [mozilla/hubs#1670]
As a user, it would be easier for me to not have to remember as many shortcuts or functionalities as possible. In this scenario, it could be helpful to see that when I hover over an object that I can press spacebar to open the object's menu as done in video-games such as Breath of the Wild.

**Comment by misslivirose** [comment]
I don't think this should show up on every hover on every object, but I think that this is the more important message to showcase in the FTUE flow / when users have 'Show Tips' selected. Right now, when the help tips show, hovering over an object shows the 'Scroll to move object closer or further from you' message, and I think it should show the 'Hold space bar' message instead on hover after the user has gone through the initial object tips.
